frontends/PyCDE/src/pycde/module.py

```diff
@@ -2,7 +2,7 @@
 #  See https://llvm.org/LICENSE.txt for license information.
 #  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-from __future__ import annotations
```

Comment:
Please keep this, it is needed to support the `dict[...]` and `list[...]` type hint syntax on Python 3.7 and 3.8.

Reply:
I was wondering why that was there...